tests/rundoctest.py

```diff
@@ -8,12 +8,18 @@
 path = os.path.dirname(__file__)
 if path:
     os.chdir(path)
+os.chdir('..')
```

Comment:
Oh I see. There might have been a trouble if cypari2 was not installed but this is impossible since at the top of the file we have `import cypari2`.